test/karma.conf-ci.js

```diff
@@ -3,8 +3,15 @@ var uuid = require('uuid');
 
 module.exports = function(config) {
 
+  coverageFiles = {
+    '*/j*/!(raw*).js': 'coverage'
+  };
+
   // If the script is not being executed from the testing directory
   if(process.cwd() !== __dirname) {
+    coverageFiles = {
+      '../scripts/*.js': 'coverage'
```

Comment:
I like this approach more than my other suggestion, but you could improve it by getting the path to the directory and checking for privly-safari. You should also leave a comment above this block pointing to the Safari repository.